openedx/core/lib/courses.py

```diff
@@ -32,6 +33,13 @@ def course_image_url(course, image_key='course_image'):
         url = settings.STATIC_URL + settings.DEFAULT_COURSE_ABOUT_IMAGE_URL
     else:
         loc = StaticContent.compute_location(course.id, getattr(course, image_key))
+        if getattr(course, image_key).endswith('images_course_image.jpg'):
+            try:
+                AssetManager.find(loc)
+            except NotFoundError:
+                url = '/static/' + settings.DEFAULT_COURSE_ABOUT_IMAGE_URL
```

Comment:
Can we use some static utility to make this path instead of hard coding it?

Reply:
I just copied the edX convention followed in the same method.
Take a look at this

https://github.com/colaraz/edx-platform/blob/0c853d5f8d9518a79f8a7c4069a5eeca3714e663/openedx/core/lib/courses.py#L26

Reply:
As we all know, there are multiple approaches to a solution. The current one is good and there is no issue with it. 
Just have a look at this answer too.
https://stackoverflow.com/questions/11721818/django-get-the-static-files-url-in-view

Reply:
BTW it's optional and I've approved the PR